token/program-2022/src/extension/mod.rs

```diff
@@ -272,8 +273,9 @@ fn check_account_type<S: BaseState>(account_type: AccountType) -> Result<(), Pro
 ///                          ^     ^       ^     ^
 ///                     acct type  extension length data...
 ///
-/// Mint: 82 bytes... + 83 bytes of other extension data + [2, 0, 3, 0, 100, ....]
-///                                                         ^ data in extension just happens to look like this
+/// Mint: 82 bytes... + 83 bytes of other extension data + [2, 0, 3, 0, 100,
+/// ....]                                                         ^ data in
+/// extension just happens to look like this
```

Comment:
Eh, this is not good. might need to do something like
```suggestion
/// Mint: 82 bytes... + 83 bytes of other extension data + 
/// [2, 0, 3, 0, 100, ....]
///       ^ data in extension just happens to look like this
```